l10n_ch_fds_upload_sepa/wizards/fds_postfinance_file_pain000.py

```diff
@@ -0,0 +1,45 @@
+from odoo import models, api
+
+import base64
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class FdsPostfinanceFilePain000(models.Model):
+    _inherit = 'fds.postfinance.file'
+
+    @api.multi
+    def import2bankStatements(self):
+        res = True
+        account_pain000 = self.env['account.pain000.parser']
+        for pf_file in self:
+            values = {'data_file': pf_file.data}
+            bs_import_obj = self.env['account.bank.statement.import']
+            bank_wiz_imp = bs_import_obj.create(values)
+
+            decoded_file = base64.b64decode(bank_wiz_imp.data_file)
+            result = account_pain000.parse(decoded_file)
```

Comment:
This could fail if the file is not a pain file. You should surround with try/except ValueError:
In the except clause you can just call continue to go to the next file.